src/e2e/app/cli/repo-tests/src/test_smtp.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use kamu_cli::testing::Kamu;
+use kamu_cli_e2e_common::KamuApiServerClient;
+use reqwest::Url;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_smpt_push_pull_sequence(kamu_api_server_client: KamuApiServerClient) {
```

Comment:
SMTP associates with emailing, and exactly here, it's misspelled.
Maybe we should not use such abbreviation, could be simply "smart push", "smart pull".

Reply:
Updated -- done